cw-orch/src/lib.rs

```diff
@@ -6,37 +6,49 @@
 pub use cw_orch_contract_derive::interface;
 pub use cw_orch_fns_derive::{ExecuteFns, QueryFns};
 
-/// Re-export anyhow for use in the macros
-pub extern crate anyhow;
-
+#[cfg(not(target_arch = "wasm32"))]
 // prelude
 pub mod prelude;
 
-pub use cw_orch_core::build;
-pub use cw_orch_core::contract;
-
-/// Related to execution environents and variables
-pub mod environment {
-    pub use cw_orch_core::env::{default_state_folder, CwOrchEnvVars};
-    pub use cw_orch_core::environment::*;
-}
-pub use cw_orch_mock as mock;
-
-/// Re-export tokio, the async runtime when using daemons.
-#[cfg(feature = "daemon")]
-pub extern crate tokio;
+#[cfg(not(target_arch = "wasm32"))]
 #[cfg(feature = "daemon")]
 pub mod daemon;
 
+#[cfg(not(target_arch = "wasm32"))]
 mod error;
 #[cfg(feature = "osmosis-test-tube")]
 pub mod osmosis_test_tube;
```

Comment:
We should either arch-flag all the exports or only the non-featured ones (if we assume a user doesn't enable the feature in their wasm env). 

Right now daemon is arch-flagged but test-tube is not.

Reply:
I think arch-flagging features would work as well!

Reply:
Oups, that's a mistake yes

Reply:
flaggin features doesn't seem to work : https://github.com/rust-lang/cargo/issues/1197